SharedSource/Core/Utilities/MemberInfoFromExpressionUtility.cs

```diff
@@ -202,7 +202,7 @@ private static ConstructorInfo GetConstructorInfoFromNewExpression (Expression e
       if (newExpression == null)
         throw new ArgumentException ("Must be a NewExpression.", "expression");
 
-      return newExpression.Constructor;
+      return newExpression.Constructor!;
```

Comment:
you really do have to wonder how this is possible.

Reply:
Ctor is `null` for `NewValueTypeExpression` instances.

Reply:
In theory, we should consider adding a null-guard here, then.

Reply:
Separate ticket?